ProjectLighthouse/Pages/ExternalAuth/AuthenticationPage.cshtml

```diff
@@ -14,22 +14,48 @@
 else
 {
     <p>You have @Model.AuthenticationAttempts.Count authentication attempts pending.</p>
-    <a href="/authentication/denyAll">
-        <button class="ui small red button">Deny all</button>
-    </a>
+    @if (Model.IpAddress != null)
+    {
+        <p>This device's IP address is <b>@(Model.IpAddress.ToString())</b>. If this matches with an authentication attempt below, then it's safe to assume the authentication attempt came from the same network as this device.</p>
+    }
 }
 
+<a href="/authentication/autoApprovals">
+    <button class="ui small blue button">
+        <i class="cog icon"></i>
+        <span>Manage automatically approved IP addresses</span>
+    </button>
+</a>
+<a href="/authentication/denyAll">
+    <button class="ui small red button">
+        <i class="x icon"></i>
+        <span>Deny all</span>
+    </button>
+</a>
+
 @foreach (AuthenticationAttempt authAttempt in Model.AuthenticationAttempts)
 {
     DateTimeOffset timestamp = DateTimeOffset.FromUnixTimeSeconds(authAttempt.Timestamp);
     <div class="ui red segment">
         <p>A <b>@authAttempt.Platform</b> authentication request was logged at <b>@timestamp.ToString("MM/dd/yyyy @ h:mm tt") UTC</b> from the IP address <b>@authAttempt.IPAddress</b>.</p>
```

Comment:
I know this isn't really something new, but I'm wondering if we should maybe move the `MM/dd/yyyy @ h:mm tt` date string into a title attribute so the user can see the absolute time when hovering, and then have something more human readable for glancing:

(i.e. the user sees "A request was logged **a few seconds ago** from the IP address [...]" etc.

I know there are JS libraries that do this relative time wording like  [momentjs](https://momentjs.com/docs/#/customization/relative-time/), but there may be something similar that's better suited for this project.

Furthermore, we might want to have a little [!] warning indicator somewhere here if the IP address doesn't match the one shown up top, just to bring attention to the fact that the request may have come from outside their network. Geting even fancier, maybe we can include the approximate region/country of the request? Not sure what it takes to get that kind of info, but it definitely helps someone put more pieces of the puzzle together if they aren't familiar with IP addresses.